### PR TITLE
Say that the static package is only usable as a prod env

### PR DESCRIPTION
**Pull request by oupala**

### CODE REVIEW

File: en/admin/installation/installation.md [hunk 1]
```diff
@@ -30,7 +30,9 @@ If you're installing Wallabag behind Squid as a reverse proxy, make sure to upda
 ## On shared hosting
 
 We provide a package with all dependencies inside. The default
-configuration uses MySQL (SQLite is no longer supported from version 2.4) for the database. To add the setting for your database, please edit `app/config/parameters.yml`. Beware that passwords must be surrounded by single quotes ( ' ).
+configuration uses SQLite for the database. To add the setting for your database, please edit `app/config/parameters.yml`. Beware that passwords must be surrounded by single quotes (').
```
Comment (j0k3r) [inline en/admin/installation/installation.md:33]: Nope, the default is MySQL
```suggestion
configuration uses MySQL for the database. To add the setting for your database, please edit `app/config/parameters.yml`. Beware that passwords must be surrounded by single quotes (').
```

File: en/admin/installation/installation.md [hunk 2]
```diff
@@ -45,6 +47,8 @@ wget https://wllbg.org/latest-v2-package && tar xvf latest-v2-package
 You will find the [md5 hash of the latest package on our
 website](https://wallabag.org/en#download).
 
+The static package requires each command to be appended by --env=prod as the static package is only usable as a prod environment (dev environment is not supported and won't work at all).
```
Comment (j0k3r) [inline en/admin/installation/installation.md:50]:
```suggestion
The static package requires each command to be appended by `--env=prod` as the static package is only usable as a prod environment (dev environment is not supported and won't work at all).
```

File: en/admin/installation/installation.md (hunk 1 above)
Comment (oupala) [reply en/admin/installation/installation.md]: Change rebased in last force push.

File: en/admin/installation/installation.md (hunk 2 above)
Comment (oupala) [reply en/admin/installation/installation.md]: Change rebased in last force push.

**Review by j0k3r — APPROVED** [review]

**Pull request merged**